### DISCUSSION TITLE
Hiding relationship titles in views

### DISCUSSION THREAD

**Discussion by susliko** [likec4/likec4#535] [Q&A] [answered]
Hi!

Use Case: relationship titles are cluttering a high-level System context view, but are useful at a Container level

Is it possible to hide them for a specific view, under the `include *` statement?

e.g.
```
view systemContext of system {
  include *->* with { // like we are customizing some specific element
    title ''
  }
}
```

**Comment by davydkov** [comment, marked as answer]
Implemented with #910

**Reply by susliko** [reply]
Thank you!
